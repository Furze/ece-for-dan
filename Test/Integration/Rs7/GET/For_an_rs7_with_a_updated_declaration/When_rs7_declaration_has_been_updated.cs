using Bard;
using MoE.ECE.Domain.Read.Model.Rs7;
using MoE.ECE.Integration.Tests.Chapter;
using MoE.ECE.Integration.Tests.Infrastructure;
using Shouldly;
using Xunit;
using Xunit.Abstractions;

namespace MoE.ECE.Integration.Tests.Rs7.GET.For_an_rs7_with_a_updated_declaration
{
    public class When_rs7_declaration_has_been_updated : SpeedyIntegrationTestBase
    {
        public When_rs7_declaration_has_been_updated(RunOnceBeforeAllTests testSetUp, ITestOutputHelper output,
            TestState<ECEStoryBook, ECEStoryData> testState) : base(testSetUp, output, testState)
        {
        }

        private const string Url = "api/rs7";

        private Rs7Model Rs7Model
        {
            get => TestData.Rs7Model;
            set => TestData.Rs7Model = value;
        }

        protected override void Arrange()
        {
            Given
                .A_rs7_has_been_created()
                .The_rs7_has_been_submitted_for_peer_approval()
                .The_rs7_declaration_has_been_updated()
                .GetResult(storyData => Rs7Model = storyData.Rs7Model);
        }

        protected override void Act()
        {
            When.Get($"{Url}/{Rs7Model.Id}");
        }

        [Fact]
        public void Declaration_has_been_updated()
        {
            var response = Then
                .Response
                .Content<Rs7Model>();

            response.Declaration?.Role.ShouldBe("role");
            response.Declaration?.ContactPhone.ShouldBe("123");
            response.Declaration?.FullName.ShouldBe("joe bloggs");
        }
    }
}