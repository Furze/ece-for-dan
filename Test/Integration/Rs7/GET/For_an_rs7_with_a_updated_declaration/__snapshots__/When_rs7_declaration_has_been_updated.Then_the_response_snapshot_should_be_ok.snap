{
  "Id": 177090,
  "BusinessEntityId": "090f7f7f-eab6-4bb3-9d45-82d0cfd94db1",
  "OrganisationId": 114895,
  "FundingPeriod": "July",
  "FundingYear": 2021,
  "FundingPeriodYear": 2020,
  "IsZeroReturn": false,
  "RollStatus": "ExternalSubmittedForApproval",
  "ReceivedDate": null,
  "RevisionNumber": 1,
  "RevisionDate": "2020-11-12T14:59:02.344434+13:00",
  "AdvanceMonths": [
    {
      "Id": 0,
      "MonthNumber": 7,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "Id": 0,
      "MonthNumber": 8,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "Id": 0,
      "MonthNumber": 9,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "Id": 0,
      "MonthNumber": 10,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    }
  ],
  "EntitlementMonths": [
    {
      "Id": 0,
      "MonthNumber": 2,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    },
    {
      "Id": 0,
      "MonthNumber": 3,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 30,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 31,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    },
    {
      "Id": 0,
      "MonthNumber": 4,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 30,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    },
    {
      "Id": 0,
      "MonthNumber": 5,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 30,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 31,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    }
  ],
  "IsAttested": true,
  "Declaration": null,
  "RequestId": "177090"
}
